ramiro_serra/dif_driver_investigation.md

```diff
@@ -0,0 +1,53 @@
+# Differential wheeled robots
```

Comment:
It's the first time I hear the term "differential wheeled". I always use "differential-drive robot". Anyways, it's okay.